randovania/exporter/hints/joke_hints.py

```diff
@@ -34,4 +34,11 @@
     "What is a hint? A miserable little pile of secrets!",
     "They say that not all hints are useful.",
     "You get nothing! You lose! Good day, sir!",
+    "Your next line is: 'Urgh, a joke hint.'",
+    "All your hints are belong to us.",
+    "A joke hint? In my randomized game? It's more likely than you think!",
+    "A wild joke hint appeared!",
+    "Shorts are comfy and easy to wear.",
+    "Nobody expects a joke hint! Its chief weapon is surprise... and fear.",
+    "Randovania's first release was on the 10th October of 2017.",
```

Comment:
the date format here is unusual. I know there are a lot of different standards for date formats in English but I'm not aware of one where this format is valid lol

Reply:
oh, its "10th of October 2017", isnt it?